app/boards/arm/bt60/bt60_v1.keymap

```diff
@@ -2,33 +2,9 @@
 #include <dt-bindings/zmk/keys.h>
 #include <dt-bindings/zmk/bt.h>
 
-#define ANSI true
-//#define HHKB true
-//#define ISO true
-//#define ALL_1U true
-//#define SPLIT_BKSP_RSHFT true
-
-
-
 / {
-    chosen {
-    #ifdef ANSI
-        zmk,matrix-transform = &ansi_transform;
-    #elif defined(HHKB)
-        zmk,matrix-transform = &hhkb_transform;
-    #elif defined(ISO)
-        zmk,matrix-transform = &iso_transform;
-    #elif defined(ALL_1U)
-        zmk,matrix-transform = &all_1u_transform;
-    #else
-        zmk,matrix-transform = &split_transform;
-    #endif
-    };
```

Comment:
It may be nice to preserve this for non-studio builds, but converted to select the physical layout?

Reply:
Good idea. I'll incorporate this